edgelet/contrib/config/linux/config.yaml

```diff
@@ -127,8 +127,8 @@ hostname: "<ADD HOSTNAME HERE>"
 ###############################################################################
 
 connect:
-  management_uri: "unix:///run/aziot/edged/aziot-edged.mgmt.sock"
-  workload_uri: "unix:///run/aziot/edged/aziot-edged.workload.sock"
+  management_uri: "unix:///var/lib/aziot/edged/aziot-edged.mgmt.sock"
```

Comment:
What's the reason to move them back to homedir?

Reply:
I was going through contrib/centos/aziot-edge.spec and the sockets are created in %{iotedge_home}. I think they were only moved out on debian.

Reply:
As I said on Teams:

>iotedged used to bind those sockets under its homedir, ie `/var/lib/iotedge` in that case. I think Rajeev changed it to be `/run/aziot/edged`, ie the same as if it was using socket activation (which should be fine)

If `RuntimeDirectory=` works on CentOS, there is no reason to move them back to the homedir.

Reply:
Also:

>I was going through contrib/centos/aziot-edge.spec and the sockets are created in %{iotedge_home}.

That is pointless. Even if the socket files are created up-front they're not usable as sockets. Not only are they created as text files, even if they were created as socket files they would need to be deleted first because `bind()` needs to create them, which is what the code already does. Just delete that from the spec file.

Reply:
RuntimeDirectory= doesn't work on CentOS 7. So I'll have to try something else

Since iotedge is hardcoded to use "unix:///run/aziot/edged/aziot-edged.mgmt.sock" I'll have to change this anyways.

Reply:
Okay, that's what I was looking for.

In that case, yes, the socket should go in homedir.

Reply:
You also need to fix https://github.com/Azure/iotedge/blob/iiot/edgelet/Makefile#L99 so that `iotedge` uses the right default.

Reply:
Done. I've changed the sockets to be in /var/lib.

Also, it seems like creating /var/lib/aziot/edged/aziot-edge.*.sock is necessary in the .spec file. Removing it causes rpmbuild to fail: Installed (but unpackaged) file(s) found

Reply:
Because they're being packaged. https://github.com/Azure/iotedge/blob/iiot/edgelet/Makefile#L131-L132 Don't package them.

Reply:
Fixed

Reply:
Ahh, the reason this was originally done is because the code will not bind the sockets with the right permissions otherwise - https://github.com/Azure/iotedge/blob/iiot/edgelet/edgelet-http/src/unix.rs#L20 If the socket file already exists, then the existing mode is preserved. But if the file doesn't exist, then it's bound with the default umask.

That'll probably make the sockets reachable to all users, which is fine for workload but not for mgmt.

Threading down the permissions through the call chain is not trivial. For now, remove your last commit please.

Reply:
Done